compiler/src/dmd/e2ir.d

```diff
@@ -7021,20 +7021,26 @@ elem *callCAssert(ref IRState irs, const ref Loc loc, Expression exp, Expression
     }
     else
     {
-        version (CRuntime_Musl)
-        {
-            // __assert_fail(exp, file, line, func);
-            elem* efunc = getFuncName();
-            auto eassert = el_var(getRtlsym(RTLSYM.C__ASSERT_FAIL));
-            ea = el_bin(OPcall, TYvoid, eassert, el_params(elmsg, efilename, eline, efunc, null));
-        }
-        else
-        {
-            // [_]_assert(msg, file, line);
-            const rtlsym = (irs.target.os == Target.OS.Windows) ? RTLSYM.C_ASSERT : RTLSYM.C__ASSERT;
-            auto eassert = el_var(getRtlsym(rtlsym));
-            ea = el_bin(OPcall, TYvoid, eassert, el_params(eline, efilename, elmsg, null));
+        Symbol *assertSym;
+        elem* params;
+        with (TargetC.Runtime) switch (irs.target.c.runtime)
+        {
+            case Musl:
+            case Glibc:
+                // __assert_fail(exp, file, line, func);
+                assertSym = getRtlsym(RTLSYM.C__ASSERT_FAIL);
+                elem* efunc = getFuncName();
+                params = el_params(efunc, eline, efilename, elmsg, null);
+                break;
+            default:
+                // [_]_assert(msg, file, line);
+                const rtlsym = (irs.target.os == Target.OS.Windows) ? RTLSYM.C_ASSERT : RTLSYM.C__ASSERT;
+                assertSym = getRtlsym(rtlsym);
+                params = el_params(eline, efilename, elmsg, null);
+                break;
         }
+        auto eassert = el_var(getRtlsym(RTLSYM.C__ASSERT_FAIL));
+        ea = el_bin(OPcall, TYvoid, eassert, params);
```

Comment:
```suggestion
        auto eassert = el_var(assertSym);
        ea = el_bin(OPcall, TYvoid, eassert, params);
```

Looks like just a small typo.

Reply:
Good call! I did not mean to leave that in there.